frontline-functions-quickstart/assets/providers/customers.private.js

```diff
@@ -0,0 +1,142 @@
+/* eslint-disable camelcase */
```

Comment:
why is this required?

Reply:
Because otherwise it complains about `customer_id` and `display_name` on the customer object